account-kit/core/src/actions/createAccount.ts

```diff
@@ -112,6 +123,15 @@ export async function createAccount<TAccount extends SupportedAccountTypes>(
           signer,
           transport: (opts) => transport({ ...opts, retryCount: 0 }),
           chain,
+        }).then((account) => {
```

Comment:
Question? Why not put this directly in the createXXXXAccount functions the smart accounts sub package?

Reply:
This particular function is used to create accounts leveraging our signer and handles writing to state as well. Later will make up part of the API for one contract

Reply:
OH I'm dumb. I see what you mean